manifests/m/mellobacon/scpterminal/0.0.2.0/mellobacon.scpterminal.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: mellobacon.scpterminal
+PackageVersion: 0.0.2.0

```

Hmmm, unluckily for this program the ARP table says version 0.0.1.0
```suggestion
PackageVersion: 0.0.1.0
```